state.tf

```diff
@@ -8,9 +8,9 @@ data "terraform_remote_state" "core_apps_infrastructure" {
   backend = "azurerm"
 
   config {
-    resource_group_name  = "mgmt-state-store-${var.subscription}"
-    storage_account_name = "mgmtstatestore${var.subscription}"
-    container_name       = "mgmtstatestorecontainer${var.env}"
+    resource_group_name  = join("-",["mgmt-state-store",var.subscription])
+    storage_account_name = join("",["mgmtstatestore",var.subscription])
```

Comment:
better to remove join changes from line 12,13 pls, as it's not having any concatenation char like '-'

Reply:
removed